fsck/fsck.c

```diff
@@ -1190,6 +1194,9 @@ static int exfat_root_dir_check(struct exfat *exfat)
 	exfat_debug("root directory: start cluster[0x%x] size[0x%" PRIx64 "]\n",
 		root->first_clus, root->size);
 
+	if (read_volume_label(exfat))
```

Comment:
Can you explain why read_volume_label() move to here ? There is no reason in patch description.
And can you check patch description in other patches also ?